accurest-core/src/test/groovy/io/codearte/accurest/builder/JaxRsClientSpockMethodBuilderSpec.groovy

```diff
@@ -26,8 +26,8 @@ class JaxRsClientSpockMethodBuilderSpec extends Specification {
 		when:
 			builder.appendTo(blockBuilder)
 		then:
-			blockBuilder.toString().contains("responseBody.property1 == '''a'''")
-			blockBuilder.toString().contains("responseBody.property2 == '''b'''")
+			blockBuilder.toString().contains("\$[?(@.property1 == 'a')]")
```

Comment:
I'm just thinking - it's pretty context syntax. Maybe we could use some simple wrapper in tests, which will internally assert json paths. Like:
```
blockBuilder.toString().contains("body.property1 == 'a'")
```